cypress/support/index.ts

```diff
@@ -28,6 +28,10 @@ declare global {
 				selector: string,
 				...args: Array<Partial<Loggable & Timeoutable & Withinable & Shadow> | undefined>
 			): Chainable<JQuery<HTMLElement>>;
+			ifCanvasVersion(
+				getterV1: () => Chainable<JQuery<HTMLElement>>,
+				getterV2: () => Chainable<JQuery<HTMLElement>>,
+			): Chainable<JQuery<HTMLElement>>;
```

Comment:
A more generic approach maybe?

```suggestion
			ifCanvasVersion<T1, T2>(
				getterV1: () => T1,
				getterV2: () => T2,
			): T1 | T2;
```